query-graphs-js/src/pathTree.ts

```diff
@@ -222,7 +231,7 @@ export class PathTree<TTrigger, RV extends Vertex = Vertex, TNullEdge extends nu
       }
     }
     assert(idx === totalChilds, () => `Expected to have ${totalChilds} childs but only ${idx} added`);
-    return new PathTree<TTrigger, RV, TNullEdge>(graph, currentVertex, localSelections, triggerEquality, childs);
+    return new PathTree<TTrigger, RV, TNullEdge>(graph, currentVertex, localSelections, triggerEquality, childs, null, null);
```

Comment:
You can delete the function this code is a part of (`mergeAllTreesInternal()`) and its only caller (`mergeAllOpTrees()`), as they're both dead code.